### PR TITLE
[RFC][UN-7174] Added footer component

### PR DESCRIPTION
**Pull request by asantos00**
Dependent of https://github.com/uniplaces/uniplaces-uniq/pull/41/files

https://uniplaces.atlassian.net/browse/UN-7174

### CODE REVIEW

File: app/components/uni-footer-website.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export { default } from 'ember-cli-uniq/components/uni-footer-website';
+
```
Comment (montoias) [inline app/components/uni-footer-website.js]: remove extra line

File: addon/templates/components/uni-footer-website.hbs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="uni-footer-website__wrapper">
+    {{yield (hash row=(component "uni-footer-website-row"))}}
```
Comment (montoias) [inline addon/templates/components/uni-footer-website.hbs]: don't hardcode it here.  Have it defined as a variable in uni-footer-website. 

That way you can redefine it

File: addon/templates/components/uni-footer-website-row.hbs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{{yield (hash cell=(component "uni-footer-website-row-cell"))}}
+
```
Comment (montoias) [inline addon/templates/components/uni-footer-website-row.hbs]: empty line

File: addon/templates/components/uni-footer-website-row.hbs [hunk 4]
```diff
@@ -0,0 +1,2 @@
+{{yield (hash cell=(component "uni-footer-website-row-cell"))}}
```
Comment (montoias) [inline addon/templates/components/uni-footer-website-row.hbs]: same

File: addon/templates/components/uni-footer-website-row-cell.hbs [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{{#if header}}
+    <div class="uni-footer-website__row__cell__header">{{header}}</div>
+{{/if}}
+{{yield this}}
```
Comment (montoias) [inline addon/templates/components/uni-footer-website-row-cell.hbs]: new line

File: addon/templates/components/uni-footer-website-row-cell.hbs [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (montoias) [inline addon/templates/components/uni-footer-website-row-cell.hbs:4]: empty line

File: addon/components/uni-footer-website-row.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+import layout from '../templates/components/uni-footer-website-row';
+
+import cell from './uni-footer-website-row-cell';
+
+const { Component } = Ember;
+
+export default Component.extend({
+    classNames: ['uni-footer-website__row'],
+    layout,
+    cell
```
Comment (montoias) [inline addon/components/uni-footer-website-row.js]: new line

Comment (montoias) [reply addon/components/uni-footer-website-row.js]: do 

```js
export default Component.extend({
    classNames: ['uni-footer-website__row'],
    layout,

    cell
})
```

File: addon/components/uni-footer-website.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+import layout from '../templates/components/uni-footer-website';
+
+import row from './uni-footer-website-row';
+
+const { Component } = Ember;
+
+export default Component.extend({
+    classNames: ['uni-footer-website'],
+    layout,
+    row,
```
Comment (montoias) [inline addon/components/uni-footer-website.js]: same here

**Review by montoias — CHANGES_REQUESTED** [review]

**Comment by asantos00** [comment]
@montoias , can you re-check? Thanks!

**Comment by asantos00** [comment]
After uniq's PR is merged we have to update version on this bower.json file

File: addon/styles/partials/_uni-footer-website.scss [hunk 9]
```diff
@@ -0,0 +1,13 @@
+.uni-footer-website {
+    @include footer-website();
+
+    &__row {
+        &__cell {
+            &__anchor {
+                &--inline {
```
Comment (mmelo) [inline addon/styles/partials/_uni-footer-website.scss]: being this just a single selector nesting and being this an exceptional case I would collapse into a single selector.

File: addon/templates/components/uni-footer-website-row-cell.hbs [hunk 10]
```diff
@@ -0,0 +1,5 @@
+{{#if header}}
+    <div class="uni-footer-website__row__cell__header">{{header}}</div>
```
Comment (mmelo) [inline addon/templates/components/uni-footer-website-row-cell.hbs]: replace `{{header}}` with `{{{header}}}` as we can imagine that further on we may want to add more than just text here (like an icon/svg) this is just to be safe

**Review by mmelo — CHANGES_REQUESTED** [review]
overall great job 💯

File: addon/templates/components/uni-footer-website-row-cell.hbs (hunk 10 above)
Comment (asantos00) [reply addon/templates/components/uni-footer-website-row-cell.hbs]: well spotted! thanks!

**Review by mmelo — APPROVED** [review]

**Review by montoias — APPROVED** [review]

**Pull request merged**